R/zchunk_batch_electricity_USA_xml.R

```diff
@@ -10,76 +10,70 @@
 #' original data system was \code{batch_electricity_USA_xml.R} (gcamusa XML).
 module_gcamusa_batch_electricity_USA_xml_DISABLED <- function(command, ...) {
```

Comment:
@bpbond I made several changes for the batch, but I'm can't enter the debug mode. So it is still disabled.